install-php-locally.md

```diff
@@ -0,0 +1,177 @@
+# Installing PHP and Symfony CLI Locally
+
+The [Symfony CLI](https://symfony.com/doc/current/setup/symfony_cli.html) is awesome for developing Symfony
+applications locally! It enables you to run a local web server, integrates with Docker containers for
+services like a database, and provides many other features to streamline your development workflow.
+
+Here at SymfonyCasts, we use it for all our Symfony courses so to best follow along, you'll want to
+install it as well.
+
+It's *mostly* batteries-included, but it does need PHP available. The official Symfony recommendation is to
+install PHP locally and natively on your computer. Let's look at how to install PHP and the Symfony CLI on
+different operating systems. We'll also make sure extensions for the most common databases (MySQL, SQLite,
+PostgreSQL) are installed.
+
+To get started, follow the instructions below for your operating system:
+1. [macOS](#macos-homebrew)
+2. [Ubuntu/Debian Linux](#ubuntudebian-linux-apt)
+3. [Windows](#windows-wsl)
+
+Once you have PHP and the Symfony CLI installed, [verify your installation](#verify-phpsymfony-cli-installation).
+
+## macOS (Homebrew)
+
+First, ensure you have the [Homebrew](https://brew.sh) package manager installed (if it isn't already):
+
+```terminal
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+### Homebrew PHP
+
+Now, install PHP:
+
+```terminal
+brew install php
+```
+
+***TIP
+This will install the latest stable version of PHP. If you want a specific version, check
+[the Homebrew PHP formula](https://formulae.brew.sh/formula/php). Under "Other versions"
+you'll find available alternate versions. For example, to install PHP 8.3, run:
+
+```terminal
+brew install php@8.3
+```
+***
+
+### Homebrew PHP Extensions
+
+Homebrew's PHP formula includes many extensions out of the box (including common database extensions).
+If you need additional extensions, use [PECL](https://pecl.php.net/) (PHP Extension Community
+Library), which was installed with PHP. For instance, to install the Redis extension, run:
+
+```terminal
+pecl install redis
+```
+
+### Homebrew Symfony CLI
+
+There are several methods to [install the Symfony CLI](https://symfony.com/download), since we already
+have Homebrew installed, let's use it to install the Symfony CLI:
+
+```terminal
+brew install symfony-cli/tap/symfony-cli
+```
+
+You should be ready to go on your Mac! [Verify the installation below](#verify-phpsymfony-cli-installation).
```

Comment:
IMO, better to link to the official website on it too. But I see your point, if we really want to make this more opinionated - I'm fine with it